charts/k8ssandra/templates/reaper/reaper.yaml

```diff
@@ -32,4 +32,22 @@ spec:
   {{- if .Values.reaper.tolerations }}
   tolerations: {{ toYaml .Values.reaper.tolerations | nindent 4 }}
   {{- end }}
+  schemaInitContainerConfig:
+    {{- if (and .Values.reaper.schemaInitContainerConfig .Values.reaper.schemaInitContainerConfig.securityContext) }}
+    securityContext: {{ toYaml .Values.reaper.schemaInitContainerConfig.securityContext | nindent 6 -}}
+    {{- else }}
+    securityContext: {{ "readOnlyRootFilesystem: true" | nindent 6 }}
+    {{- end }}
+
+  {{- if .Values.reaper.podSecurityContext }}
+  podSecurityContext: {{ toYaml .Values.reaper.podSecurityContext | nindent 4 -}}
+  {{- else }}
+  podSecurityContext: {}
+  {{- end }}
+
+  {{- if .Values.reaper.securityContext }}
```

Comment:
Could you have used the `ternary` function here like you did elsewhere?

Reply:
I will take a look for the pod security context, as this may be better as a ternary operation.

Reply:
Added ternary in latest push.